features/promotion/applying_catalog_promotions/applying_all_available_types_catalog_promotions.feature

```diff
@@ -0,0 +1,32 @@
+@applying_catalog_promotions
+Feature: Applying all available catalog promotions
+    In order to be attracted to products
+    As a Shop Owner
+    I want to see discounted products in the catalog
+
+    Background:
+        Given the store operates on a channel named "Web-US"
+        And the store has "Clothes" taxonomy
+        And the store has "Dishes" taxonomy
```

Comment:
`the store classifies its products as "Clothes" and "Dishes"` instead of these 2 steps